all-clouds/all-clouds-explanation/ubuntu-base-and-minimal-images.rst

```diff
@@ -0,0 +1,48 @@
+Ubuntu Base and Minimal Images
+==============================
+
+Broadly speaking, CPC produces two types of images - a base image and a minimal
+image.
+
+What are base and minimal images?
+---------------------------------
+
+Base images are those that are directly derived from Ubuntu Server-like images
+and are meant for general consumption. On the other hand, minimal or minimized
+images are those that are meant for machine-to-machine interactions, and have
+some niceties meant for humans stripped out (such as manpages, translations,
+editors, etc.). A great example would be our k8s images.
+
+
+State of minimal images from Mantic onwards
+-------------------------------------------
+
+From Mantic Minotaur release onward (including Noble Numbat 24.04 LTS), we
+worked through the minimal images to make it more efficient and truly minimal
+by constructing a “cloud-minimal” seed, which is nothing but the list of basic
+packages that we need in a minimal cloud installation. The package list is then
+expanded by a program called “germinate”, which includes the dependencies of
+the listed packages in the seed. Combined, we get the “image manifest”, aka the
+list of packages you’d expect to see in the minimal cloud image.
+
```

Comment:
An important detail on mantic+ is that the minimal image does not install any `Recommends` packages during image build.